src/ImportInfoDialog.h

```diff
@@ -0,0 +1,25 @@
+#pragma once
+
+#include <wx/wx.h>
+#include "wxPanelWrapper.h"
+
+class ImportInfoDialog : public wxDialogWrapper
+{
+public:
+    ImportInfoDialog(const FilePath& filename, wxWindow* parent, wxWindowID id, const TranslatableString &title, const wxPoint& pos = wxDefaultPosition,
+                          const wxSize& size = wxDefaultSize,
+                          long style = wxDEFAULT_DIALOG_STYLE | wxRESIZE_BORDER);
+    
+    bool IsCancelled();
+    
+
+private:
+    void OnCancel(wxCommandEvent& event);
+
+    wxStaticText* m_statusText;
```

Comment:
Please don't mix code-style in a single header, we use camel-case prefixed with `m` for member variable names (except for POD structures)

Reply:
fixed